tests/spread/plugins/v2/colcon-ros2-hello/task.yaml

```diff
@@ -0,0 +1,51 @@
+summary: >-
+  Build, clean, build, modify and rebuild, and run hello
+  for ros2/colcon.
+
+kill-timeout: 30m
```

Comment:
now that I see arm64, this might need to be set to something a bit more exagerated, maybe look at the other ros tests for inspiration on the right timeout value

Reply:
set to 180m!